pkg/apis/admission/types.go

```diff
@@ -92,6 +92,12 @@ type AdmissionResponse struct {
 	// PatchType indicates the form the Patch will take. Currently we only support "JSONPatch".
 	// +optional
 	PatchType *PatchType
+	// Annotations is an unstructured key value map set by remote admission controller (e.g. error=image-blacklisted).
+	// MutatingAdmissionWebhook and ValidatingAdmissionWebhook admission controller will fill in the Annotations with
```

Comment:
replace "will fill in" with "will prefix the keys"

Reply:
Done.